### ISSUE TITLE
Probable getExpression-worker robustness issue

### ISSUE THREAD

**Issue by alakel** [MyWebIntelligence/MyWebIntelligence#118]
The oracle "list url" can not save the url beyond 1000 ? When we try , it do nothing more . Then after a few days , he bluntly delete queries. Is it really loaded in the DB in the first place ? Why does-it not post bug found ?

**Comment by DavidBruant** [comment]
I tried to make a URL list and I was able to see they were saved both from the UI and introspecting the database. getExpression tasks are created in the DB as well.
The getExpression-worker process is up but doesn't pick tasks any longer. No definitive answer yet, but the problem may come from the fact that getExpression-worker works "in chain" and if one task "never ends", the chain breaks.

The implementation was a first shot and purposefully naive, so the solution is 2 folds:
* Have an independent setInterval that looks for tasks every 10s (or any other constant)
* Add a timer to any task and make it race against getExpression.
